tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/decoders/the-user-should-see-the-message.then.js

```diff
@@ -1,14 +1,14 @@
 import { Then } from 'cypress-cucumber-preprocessor/steps';
-import { getObject } from '../../../utils/driver';
+import { getElement } from '../../../utils/driver';
 import {
   buttonRestartSelector,
   messageConfirmSaveSelector,
 } from '../../../pageobjects/wazuh-menu/decoders.page';
 
 Then('The user should see the message', () => {
-  getObject(messageConfirmSaveSelector)
+  getElement(messageConfirmSaveSelector)
     .should('have.text', 'Changes will not take effect until a restart is performed.');
```

Comment:
I open the debate, should we have this text here or in an external file, for example bricks.js?

Reply:
Yes, we should at least have an enum with these texts
also I started refactoring methods like validateElementText, and have them in the driver.js in case in the future we need to use that layer for another framework